tests/services/copy_project/test_copy_project.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+from kili.client import Kili
+from kili.services.copy_project import CopyProject
+
+
+def test_copy_project_e2e_no_assets_no_labels():
+    project_fixture = json.load(open("tests/fixtures/object_detection_project_fixture.json"))[0]
```

Comment:
you should use a "real" fixture (see integration test), otherwise the project won't be deleted if there is a failure.